src/history/errors.js

```diff
@@ -48,9 +48,12 @@ function createRouterError (from, to, type, message) {
   error.to = to
   error.type = type
 
-  const newStack = error.stack.split('\n')
-  newStack.splice(1, 2) // remove 2 last useless calls
-  error.stack = newStack.join('\n')
+  if (typeof error.stack === 'string') {
+    const newStack = error.stack.split('\n')
+    newStack.splice(1, 2) // remove 2 last useless calls
+    error.stack = newStack.join('\n')
+  }
+
```

Comment:
Let's completely remove this part, it's better to have the full stack trace when debugging
```suggestion
```